docs/modules/ROOT/pages/getting-started.adoc

```diff
@@ -84,6 +86,19 @@ gradlew eclipse
 . https://help.eclipse.org/2022-03/index.jsp?topic=%2Forg.eclipse.platform.doc.user%2Ftasks%2Ftasks-importproject.htm[Import the project to Eclipse]
 . Start creating your tests
 
+=== Visual Studio Code
+
+==== Prerequisites
+
+. Install https://marketplace.visualstudio.com/items?itemName=vividus.vividus-studiop[VIVIDUS Studio]
```

Comment:
Its 404 for the link

Reply:
BTW marketplace uses the following link: 
<img width="385" alt="image" src="https://user-images.githubusercontent.com/17816729/218698840-24b74216-4af3-4625-ba65-b88e9de34fab.png">
Maybe we could also have some sort of shortcut here?

Reply:
most likely this shortcut won't work if VS code is not installed, or installed, but system is not rebooted

Reply:
I would prefer to keep the link to marketplace so user can at least read about the extension